hooks/persistence-defectdojo/src/main/java/io/securecodebox/persistence/mapping/SecureCodeBoxFindingsToDefectDojoMapper.java

```diff
@@ -0,0 +1,84 @@
+package io.securecodebox.persistence.mapping;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.securecodebox.persistence.models.DefectDojoImportFinding;
+import io.securecodebox.persistence.models.SecureCodeBoxFinding;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.URI;
+import java.time.*;
+import java.time.format.DateTimeFormatter;
+import java.util.*;
+
+public class SecureCodeBoxFindingsToDefectDojoMapper {
+  private static final Logger LOG = LoggerFactory.getLogger(SecureCodeBoxFindingsToDefectDojoMapper.class);
+  private static final DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+  private static final ObjectWriter prettyJSONPrinter = new ObjectMapper().writerWithDefaultPrettyPrinter();
+
+  public static String fromSecureCodeboxFindingsJson(String scbFindingsJson) throws IOException {
+    LOG.debug("Converting SecureCodeBox Findings to DefectDojo Findings");
+    ObjectMapper mapper = new ObjectMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+    List<DefectDojoImportFinding> DefectDojoImportFindings = new ArrayList<>();
+    List<SecureCodeBoxFinding> secureCodeBoxFindings = mapper.readValue(scbFindingsJson, new TypeReference<>() {});
+    for (SecureCodeBoxFinding secureCodeBoxFinding : secureCodeBoxFindings){
+        DefectDojoImportFindings.add(fromSecureCodeBoxFinding(secureCodeBoxFinding));
+    }
+    // create the result where the format has to be {"findings": [finding1, findings2, ...]}
+    ObjectNode ddFindingJson = mapper.createObjectNode();
+    ArrayNode arrayNode = mapper.valueToTree(DefectDojoImportFindings);
+    ddFindingJson.putArray("findings").addAll(arrayNode);
+    return ddFindingJson.toString();
+  }
+
+  protected static DefectDojoImportFinding fromSecureCodeBoxFinding(SecureCodeBoxFinding secureCodeBoxFinding) throws JsonProcessingException {
```

Comment:
```suggestion
  /**
   * ...
   * @param secureCodeBoxFinding ...
   */
  protected static DefectDojoImportFinding fromSecureCodeBoxFinding(SecureCodeBoxFinding secureCodeBoxFinding) throws JsonProcessingException {
```
A method comment would be great :)

Reply:
Done